### PR TITLE
fix(menu-item): prevent click event when disabled (VIV-1440)

### PR DESCRIPTION
**Pull request by TaylorJ76**

### CODE REVIEW

File: libs/components/src/lib/menu-item/menu-item.scss [hunk 1]
```diff
@@ -24,7 +24,7 @@
 }
 
 :host([disabled]) {
-	cursor: not-allowed;
```
Comment (rachelbt) [inline libs/components/src/lib/menu-item/menu-item.scss:27]: why did you removed that?
In button we have the not-allowed, I think we should have it here as well

Comment (TaylorJ76) [reply libs/components/src/lib/menu-item/menu-item.scss:27]: Because by adding `pointer-events: none` stops you from being able to control the cursor.

Comment (RichardHelm) [reply libs/components/src/lib/menu-item/menu-item.scss:27]: Can't we stop the event with stop propagation instead?

Comment (TaylorJ76) [reply libs/components/src/lib/menu-item/menu-item.scss:27]: No, I tried that originally, but the event originates on the host so cannot be suppressed from inside the component (if far as I could tell).

**Review by RichardHelm — APPROVED** [review]

**Pull request merged**